WD-Firmware/psp/api_inc/psp_config.h

```diff
@@ -0,0 +1,36 @@
+/* 
+* SPDX-License-Identifier: Apache-2.0
+* Copyright 2019 Western Digital Corporation or its affiliates.
+* 
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+* 
+* http:*www.apache.org/licenses/LICENSE-2.0
+* 
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+/**
+* @file   psp_config.h
+* @author Ronen Haen
+* @date   20.05.2019
+* @brief  The file defines the psp configuration
```

Comment:
what is the meaning of psp_config ?
configuration per core ? or per configuration in the core ?